.goreleaser.yml

```diff
@@ -43,8 +42,10 @@ dockers:
         goarm: ''
         ids: ['cua']
         image_templates:
-            - "circonus/{{.ProjectName}}:{{.Tag}}-amd64"
-            - "circonus/{{.ProjectName}}:latest-amd64"
+            - "registry.hub.docker.com/circonus/{{.ProjectName}}:{{.Tag}}-amd64"
```

Comment:
The image template names in the goreleaser scripts for the other agents are not in this format. Do they all need to be changed also, are they not being released correctly?

Reply:
This has always worked just fine and the syntax is what's on the goreleaser page https://goreleaser.com/customization/docker/?h=docker#docker-images

I just released the Circonus-agent 5 days ago and had no issues.

Is this an access to the Circonus org on hub issue? I've never had an issue pushing using just the org/project syntax.

Reply:
This was required because I use podman and cri-o as opposed to docker and containerd.
Podman is daemonless -- more secure and faster, can be rootless, and takes advantage of selinux and apparmor in deeper ways than docker.
I prefer using podman and cri-o over docker and containerd for those reasons.

An image without a registry name is known as an image with an unqualified registry.
Unqualified registries make assumptions that aren't always true.
In my case, the container runtime I'm using doesn't default to having docker hub as an unqualified registry.
**Unqualified images are an anti-pattern** https://www.redhat.com/en/blog/be-careful-when-pulling-images-short-name (see Conclusion)

Using the fully qualified image name doesn't have a negative impact on compatibility with things that already work.
Building with docker will still work fine with this setup.

Reply:
Ah, ok, that wasn't clear to me. The pro version supports podman, so it is probably worth investing in it for the team. It also supports post hooks. I have it personally, but didn't use any of the features in these repos.